### PR TITLE
Remove mention of filtered by token on Managing API tokens page

### PR DESCRIPTION
**Pull request by JuanitoFatas**
We’re removing this functionality to remove another possible security risk surface area. I’ve also refreshed the screenshots to reflect the current UI.

<details>
<summary>Screenshots of this PR</summary>

CleanShot 2023-12-06 at 09 26 20@2x
CleanShot 2023-12-06 at 09 26 25@2x
CleanShot 2023-12-06 at 09 26 31@2x

</details>

### CODE REVIEW

File: pages/apis/managing_api_tokens.md [hunk 1]
```diff
@@ -42,19 +42,19 @@ All tokens that currently have access to your organization's data will be listed
 
 Click through any token to see more detailed information about its scopes and the most recent request.
 
-<%= image "all-tokens-view.png", width: 1480/2, height: 1064/2, alt: "Screenshot of the API Access Audit page displaying a list of all tokens" %>
+<%= image "all-tokens-view.png", width: 1820/2, height: 1344/2, alt: "Screenshot of the API Access Audit page displaying a list of all tokens" %>
 
-The list of tokens can be filtered by username, token value, scopes, IP address, or whether the user has admin privileges.
```
Comment (jameshill) [inline pages/apis/managing_api_tokens.md:47]: yep this looks correct

**Review by jameshill — APPROVED** [review]
This looks good to me.. will wait for @mbelton-buildkite's final comment on it

**Review by himalb001 — APPROVED** [review]
🎉🎉

**Review by mbelton-buildkite — APPROVED** [review]
Looks good to me 😊

**Pull request merged**